docker/alpine.Dockerfile

```diff
@@ -62,41 +60,26 @@ RUN mkdir -p $PS_FOLDER /tmp/unzip-ps \
   && chown -R www-data:www-data $PS_FOLDER \
   && rm -rf /tmp/prestashop.zip /tmp/unzip-ps
 
-ENV DUMP_FILE="/dump.sql"
-ADD ./assets/hydrate.sh /hydrate.sh
-RUN sh /hydrate.sh
-
-# Clean up install files
-RUN rm -rf ${PS_FOLDER}/install ${PS_FOLDER}/Install_PrestaShop.html
+# Install and configure MariaDB
+RUN adduser --system mysql; \
+  apk --no-cache add -U --no-commit-hooks --no-scripts \
+  runuser mariadb-client mariadb;
+COPY ./assets/mariadb-server.cnf /etc/my.cnf.d/mariadb-server.cnf
 
-# Clean up var and recreate cache and log directories
-RUN rm -rf "${PS_CACHE_DIR}" "${PS_LOGS_DIR}" \
-  && mkdir -p "${PS_CACHE_DIR}/prod" "${PS_CACHE_DIR}/dev" "${PS_LOGS_DIR}" \
-  && chown -R www-data:www-data "${PS_CACHE_DIR}" "${PS_LOGS_DIR}"
+# Hydrate the SQL dump
+COPY ./assets/hydrate.sh /hydrate.sh
+RUN sh /hydrate.sh
 
 # -----------------------
 # Flashlight final image
 # -----------------------
-FROM base-prestashop AS optimize-prestashop
+FROM base-prestashop AS prestashop-flashlight
```

Comment:
Are we staying as root user here?

Reply:
In the image yes, but the nginx has nginx rights, and fpm actually running PHP code drops right to www-data